projects/fsastorefrontlib/src/core/my-account/facade/consent.service.spec.ts

```diff
@@ -125,75 +181,75 @@ describe('ConsentServiceTest', () => {
         customerId: customerId,
       })
     );
-    it('should transfer cart to selected obo customer', () => {
```

Comment:
There were some formatting issues with this test, so I fixed them. Not sure how (if) it worked in the first place.